externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/KyuubiSparkUtil.scala

```diff
@@ -148,9 +148,9 @@ object KyuubiSparkUtil extends Logging {
         .invoke[AnyRef](fragment)
 
       DynMethods.builder("build")
-        .impl("javax.ws.rs.core.UriBuilder")
+        .impl("javax.ws.rs.core.UriBuilder", classOf[Array[Object]])
```

Comment:
does `Array[AnyRef]` work? if so, prefer `AnyRef` over `Object` in scala code

Reply:
It work.